src/CovidAppointmentTable.js

```diff
@@ -11,73 +11,11 @@ import HelpDialog from "./components/HelpDialog";
 import { makeStyles } from "@material-ui/core";
 import MoreInformation from "./components/MoreInformation";
 import React from "react";
-import SignUpLink, { hasSignUpLink } from "./components/SignUpLink";
+import SignUpLink from "./components/SignUpLink";
 import { sortData } from "./services/appointmentData.service";
 import StaleDataIndicator from "./components/StaleDataIndicator";
 import Typography from "@material-ui/core/Typography";
 
-const dayjs = require("dayjs");
-
-// any location with data older than this will not be displayed at all
-export const tooStaleMinutes = 60; // unit in minutes
```

Comment:
Maybe I'm not understanding but why are we removing all this code? We still want it to run in prod at least, right?

Reply:
All of this code was refactored in a previous PR to `appointmentData.service.js`.  I think it must have been accidentally reinserted into this file during a merge.